src/ducks/albums/components/AlbumToolbar.jsx

```diff
@@ -86,7 +92,8 @@ export const mapDispatchToProps = (dispatch, ownProps) => ({
       .then(() => ownProps.router.replace('albums'))
       .then(() => Alerter.success('Albums.remove_album.success', {name: album.name}))
       .catch(() => Alerter.error('Albums.remove_album.error.generic'))
-  )
+  ),
+  downloadAlbum: downloadAlbum
```

Comment:
As downloadAlbum is not an action creator (no dispatch), it isn't necessary to declare it here, you could just call downloadAlbum directly in the event handler.